server/src/main/resources/schema.sql

```diff
@@ -0,0 +1,41 @@
+ALTER TABLE teachers
```

Comment:
What is this file for?
When is the SQL used?

Reply:
This is run right after the spring data initialization completes. What's this for is discussed in the description :)

Reply:
So it's run every time the initialization is completed?
This sounds a bit overkill...

Reply:
No, it's run every time something in the schema changes that spring considers worthwhile updating for. For prod this doesn't matter anyways, as it'll only run once on startup